maestro-client/src/main/java/maestro/drivers/AndroidDriver.kt

```diff
@@ -518,6 +518,38 @@ class AndroidDriver(
         )
     }
 
+    override fun setPermissions(appId: String, permissions: Map<String, String>) {
+        val mutable = permissions.toMutableMap()
```

Comment:
Launch app fails due to this, creating a fix for this. Let's defer this PR until then

Reply:
Does it fail on creating a mutable map? That's unexpected..

Reply:
Nope, sorry pointing at wrong line. While setting permission in the line below, no worries I am going to raise a fix for that very soon against this PR.

Reply:
Are there any open action items for me on this PR now that android works?

Reply:
Integration tests

Reply:
Done (but, see other thread)